main.go

```diff
@@ -0,0 +1,14 @@
+package main
+
+import (
+	"github.com/rromodev/academy-go-q32021/controllers"
+
+	"github.com/gin-gonic/gin"
+)
+
+func main() {
+	router := gin.Default()
```

Comment:
I would probably move the router creation and endpoint definitions to their own `router` package